OpenRA.Mods.RA/Traits/GpsRemoveFrozenActor.cs

```diff
@@ -35,21 +34,31 @@ public GpsRemoveFrozenActor(Actor self, GpsRemoveFrozenActorInfo info)
 			watchers = self.World.ActorsWithTrait<GpsWatcher>().Select(w => w.Trait).ToArray();
 		}
 
-		public bool RemoveActor(Actor self, Player owner)
+		public void Tick(Actor self)
```

Comment:
We need this one to erase any frozen actors that are already dead when the GPS first activates.